code/modules/events/sinfuldemon.dm

```diff
@@ -0,0 +1,61 @@
+/datum/round_event_control/sinfuldemon
+	name = "Create Demon of Sin"
+	typepath = /datum/round_event/ghost_role/sinfuldemon
+	max_occurrences = 2 //misery loves company
+	min_players = 15
+	earliest_start = 20 MINUTES
+
+/datum/round_event/ghost_role/sinfuldemon
+	var/success_spawn = 0
```

Comment:
where exactly do you use this? it's never set to anything but 0

Reply:
No idea honestly, devil and ninja had it so I added it too just in case.